### PR TITLE
Aggregate release telemetry

### PR DESCRIPTION
**Pull request by fbertsch**
I do have a concern about overloading the database, but we use a read-replica so we can handle that later, if it happens.

### CODE REVIEW

File: dags/telemetry_aggregates.py [hunk 1]
```diff
@@ -15,11 +15,26 @@
...
-    env = {"date": "{{ ds_nodash }}"},
+    env = {
+      "date": "{{ ds_nodash }}",
+      "channels": "nightly, aurora, beta"
+    },
+    uri = "https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/run_telemetry_aggregator.sh",
+    dag = dag)
+
+release_telemetry_aggregate_view = EMRSparkOperator(
+    task_id = "telemetry_aggregate_view",
```
Comment (mreid-moz) [inline dags/telemetry_aggregates.py]: `task_id` should be unique here, maybe `"telemetry_aggregate_view_release"`?

File: dags/telemetry_aggregates.py [hunk 2]
```diff
@@ -15,11 +15,26 @@
...
+    job_name = "Telemetry Aggregate View - Release",
+    instance_count = 40,
+    execution_timeout=timedelta(hours=12),
+    env = {
+      "date": "{{ ds_nodash }}",
+      "channels": "release"
```
Comment (mreid-moz) [inline dags/telemetry_aggregates.py]: Should we include `esr` too?

Comment (fbertsch) [reply dags/telemetry_aggregates.py]: ESR isn't in the spec, and requires some changes to the aggregator. Let's confirm that's something people want and tackle that separately.

Comment (mreid-moz) [reply dags/telemetry_aggregates.py]: Sounds good to me.

**Comment by mreid-moz** [comment]
> I do have a concern about overloading the database, but we use a read-replica so we can handle that later, if it happens.

We could also set release aggregates as being "downstream" from pre-release if that becomes an issue.

**Comment by fbertsch** [comment]
@mreid-moz that would probably help, but I'm more concerned by the 40 machines from that one job simultaneously committing. I'll probably have to limit the number of parallel tasks at that stage if it becomes an issue.

**Pull request merged**